docs/blueprints/EP036.rst

```diff
@@ -130,16 +173,35 @@ This event should sent before any NApp sends its first flow to be published.
     "coloring": {"base": 0}
   }
 
-Deploying pipeline
+
+
+Endpoint API
+============
+  
+  - GET `/api/kytos/multi_table/v2/` - Get all pipelines in the database. Parameters enabled and disabled are allowed.

```

Comment:
Can we also have a GET for a specific pipeline? `GET /api/kytos/multi_table/v1/pipeline/<pipeline_id>`? Just so clients can specifically get a single pipeline too?